troi/cli.py

```diff
@@ -46,9 +50,29 @@ def playlist(patch, debug, args):
               file=sys.stderr)
         return
 
-    playlist.print()
-    print("-- generated playlist with %d recordings. Open playlist by opening playlist.html in your browser." % len(playlist.entities))
-    playlist.launch()
+    if token:
+        mbids = playlist.submit(token, created_for)
+        if len(mbids) == 1:
+            print("Submitted playlist: %s" % mbids[0])
+        else:
+            print("Submitted playlists:", mbid)
```

Comment:
should this be `mbids`?
Is this a playlist id? might be nice to prefix it with http://host/playlist for easy clicking

Reply:
Done.